configure.ac

```diff
@@ -3,6 +3,14 @@ AC_INIT([pick], [1.5.4], [pick-maintainers@calleerlandsson.com])
 AM_INIT_AUTOMAKE([subdir-objects])
 AC_CONFIG_HEADERS([config.h])
 AC_PROG_CC
+
+AC_CANONICAL_HOST
+build_linux=no
+case "${host_os}" in
+linux*) build_linux=yes ;;
+esac
+AM_CONDITIONAL([LINUX], [test "$build_linux" = "yes"])
```

Comment:
Is `$build_linux` used solely to define `LINUX` so that it can be picked up in `Makefile.am` or is it used elsewhere?